### ISSUE TITLE
prove postgres can be bounced w/o data loss

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/preservation_catalog#905]
Are there any conditions under which we should not bounce the db (e.g. because it would create data loss)

Essentially, if the database gets fooched on prod, can ops bounce it.

Workers will fail ... what cleanup would be required after this;  would we want to document this in ops concerns document.

**Comment by atz** [comment]
"Bouncing" seems an imprecise description for exploratory "proving" type work.  The database can be replicated, backed up, restored, etc.  Queue data is not dependent on postgres, so "in flight" replication would resume when workers were available.  We have error queues that are restartable.  And overall, the entire system is self correcting by virtue of rediscovery of Moabs, C2A, CV, etc.  Eventually the system will correct or at least detect any gaps.  

In short, I don't see there is anything actionable on this ticket, specifically no additional criteria to be met.
